javascript/load_table.js

```diff
@@ -30,7 +32,15 @@ dispatch.on("load_table", function (tbl_data) {
     let table = d3.select("#table_div")
                     .append('table')
                     .attr("id", "adv_tbl")
-                    .attr("class", "table table-condensed");
+                    //.attr("class", "table table-condensed");
+                    .attr("class","wb-tables table table-striped table-hover")
```

Comment:
As mentionned, the table sort does not work, we will decline for now, until it is resolved

Reply:
I think you commented right when I pushed the fixes. The issue is resolved and this version has the sorting working fine.